app/src/ui/branches/ci-check-run-list.tsx

```diff
@@ -210,13 +210,18 @@ export class CICheckRunList extends React.PureComponent<
   }
 
   private renderList = (checks: ReadonlyArray<IRefCheck>) => {
+    const baseHref =
+      this.props.repository.htmlURL !== null
+        ? new URL('/', this.props.repository.htmlURL).href
+        : null
```

Comment:
Thinking about this some more I think this is probably the more idiomatic way to get the host address

```suggestion
    const baseHref = getHTMLURL(this.props.repository.endpoint)
```